pkgs/applications/terminal-emulators/byrd/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, vte, libconfig, glib, pkgconfig }:
+
+stdenv.mkDerivation {
+  pname = "byrd";
+  version = "git-20200312";
```

Comment:
```suggestion
  version = "unstable-2020-03-12";
```

Reply:
it's actually 1.3.2 (as indicated on github)